### DISCUSSION TITLE
New object selection workflow seems to complicated

### DISCUSSION THREAD

**Discussion by v0tti** [netbox-community/netbox#13021] [General]
Our users are having a hard time using the new object selection method. I have to agree, it got pretty complicated doing basic things like assigning an IP address to an interface from the IP address form.

Prior versions: select device -> select interface -> done
Current version: open selector -> activate search for device -> select device -> hit search -> select interface -> done 

Do others have similar experiences? How about re-adding those pre-selection options for simple tasks like assigning IP addresses to interfaces (the selector could still be an option).

**Comment by abhi1693** [comment]
There is an open issue at https://github.com/netbox-community/netbox/issues/12751 if you'd like to help propose a solution
